_data/utilities.yml

```diff
@@ -179,6 +179,12 @@ websites:
       img: nicorgas.png
       tfa: No
 
+    - name: O2
+      url: http://www.o2.co.uk/
```

Comment:
Please change the url to `https://www.o2.co.uk/`

Reply:
Done